File: Atividade 1/Exercício 2.c
#include <stdio.h>
#include <stdlib.h>

int main(){
    char sexo, nome[60];
    int idade;
    printf("Digite seu nome: \n");
    scanf("%s", &nome);
    printf("Digite o sexo. M ou F: \n");
    scanf("%s", &sexo);
    printf("Digite a idade: \n");
    scanf("%d", &idade);
    if (sexo == 'm' || sexo == 'M'){
        printf("Nao eh mulher.\n");
    }
    else printf("Eh mulher.\n");
    return 0;
}

File: Atividade 1/Exercício 2.py
nome = input("Digite o nome: ");
while(True):
    sexo = input("Digite o sexo. 'M' ou 'F': ").upper();
    if sexo == 'M' or sexo == 'F':
        break;
    else:
        print("Sexo inválido, digite novamente.");
idade = int(input("Digite a idade: "));

if sexo == 'F':
    print("É mulher.");
else:
    print("Não é mulher.");